webhook_to_fedora_messaging/migrations/env.py

```diff
@@ -30,7 +29,7 @@
 # my_important_option = alembic_config.get_main_option("my_important_option")
 # ... etc.
 
-url = str(get_url_from_app(create_app))
+url = standard.sync_database_url
```

Comment:
That won't work once we have only async. The `sqlalchemy_helper` doc has an example of the changes to make to `env.py` to support async: 
- https://sqlalchemy-helpers.readthedocs.io/en/latest/async.html#alembic
- https://github.com/fedora-infra/sqlalchemy-helpers/blob/develop/docs/aio-env.py.example

(it's based on [this upstream doc](https://alembic.sqlalchemy.org/en/latest/cookbook.html#using-asyncio-with-alembic))
Note that the docs recommend using the sync drivers in the configuration, as those will be translated by the manager, and therefore you can keep using alembic in sync mode. There may be another better option, I don't find it very clean, but we can investigate that later.